.travis.yml

```diff
@@ -18,3 +18,5 @@ notifications:
       Build %{build_number}</a> on branch <i>%{branch}</i> by %{author}: <strong>%{message}</strong>
       <a href="https://github.com/%{repository}/commits/%{commit}">View on GitHub</a>'
     format: html
+
```

Comment:
It looks like this edit is causing the tests to fail. Could you please revert this file?